service/src/main/java/io/camunda/service/TenantServices.java

```diff
@@ -97,5 +98,13 @@ public CompletableFuture<?> removeMember(
             .setEntity(entityType, entityKey));
   }
 
+  public Collection<TenantEntity> getTenantsByMemberKey(final long memberKey) {
+    return search(
```

Comment:
this will only return a maximum of 100 tenants.
check-out https://github.com/camunda/camunda/blob/95fd2f72a5b0b62e32ed09de3e0c4ed701b9a669/search/search-domain/src/main/java/io/camunda/search/page/SearchQueryPage.java#L17
Maybe this could be enough for real world use cases, but we can use scrolling to fetch all tenants

Reply:
I've added a comment and increased the limit to 10k. We should have a utility to fetch all results for a query, but I don't want to add it in this PR.

Reply:
I did the same [here](https://github.com/camunda/camunda/pull/25749) for Roles 
But I agree we can do it in a separate PR, this one is starting to big quite big